lib/api/core/notifier.js

```diff
@@ -310,17 +309,10 @@ class NotifierController {
       updatedDocument;
     const
       cachePrefix = getCachePrefix(request),
-      getRequest = new Request({
-        action: 'update',
-        controller: 'document',
-        collection: request.input.resource.collection,
-        index: request.input.resource.index,
-        _id: request.input.resource._id,
-        requestId: request.id,
-        volatile: request.input.volatile
-      });
+      { index, collection, _id } = request.input.resource;
 
-    return this.storageEngine.get(getRequest)
+    // @todo do not make another get request, use the _source returned by update request
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)